README.md

```diff
@@ -147,14 +147,32 @@ Hostname                    | External Ports   | Internal Url
 If you run the `./start_ocean.sh` script with the `--local-spree-node` option (please see [Keeper Node](#keeper-node) section of this document for more details).
 You will have available a keeper node in the local and private `spree` network with the following accounts enabled:
 
-Account                                      | Password   | Balance
----------------------------------------------|------------|--------
-`0x00bd138abd70e2f00903268f3db08f2d25677c9e` | node0      | 10000000111000111000111000
-`0x068ed00cf0441e4829d9784fcbe7b9e26d4bd8d0` | secret     | 100000000
-`0xa99d43d86a0758d5632313b8fa3972b6088a21bb` | secret     | 100000000
+Account                                      | Type     | Password/Key                 | Balance
+---------------------------------------------|----------|------------------------------|-----------------
+`0x0011598De1016A350ad719D23586273804076774` | key      |                              | 1000000000 Ether
+`0x00Bd138aBD70e2F00903268F3Db08f2D25677C9e` | key      | node0                        | 1000000000 Ether
+`0x068Ed00cF0441e4829D9784fCBe7b9e26D4BD8d0` | key      | secret                       | 1000000000 Ether
+`0x6B0c56d1Ad5144b4d37fa6e27DC9afd5C2435c3B` | key      |                              | 1000000000 Ether
```

Comment:
@jcortejoso do you know the password? or shall we completely remove this account? From chain config and the key?

Reply:
`0x6B0c56d1Ad5144b4d37fa6e27DC9afd5C2435c3B` has an empty password. Also we can think about removing. It is also coming from the original ethereum-playground project.

Reply:
i removed it